node-template/Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
-name = "node-template"
+name = "template-node"
```

Comment:
This is incorrect and needs a fix in the script.

Reply:
I have opened https://github.com/paritytech/substrate-up/pull/19 for it